src/content/en/2021/caching.md

```diff
@@ -1,20 +1,488 @@
+  height=436,
+  sheets_gid="1202769738",
+  sql_file="non_cacheable_by_resource_type.sql"
+) }}
+
+However, some of our most commonly used resources are non-cacheable, likely due to their dynamic nature. Notably, HTML saw the highest percentage of non-cacheable resources at 23.4%, followed closely by Images with 10.1%.
+
+When we compare the mobile data between 2020 and 2021, we notice a 5.1% increase in cacheable HTML. This tells us we may be moving towards better usage of our CDNs to cache HTML pages, like those rendered by an SSR that tend to change less frequently. Pages are typically generated by Server-Side Rendered (SSR) applications if the content of a particular web page doesn't change frequently. The url can potentially serve the same HTML for weeks or even months, making that content highly cacheable.
+
+Taking a look at the Median TTL across all resource types, we see that even if we cache a similar percentage in total, there is a much longer cache for mobile, particularly for HTML, audio and video.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th colspan="6" scope="col">Median TTL (in days)</th>
+      </tr>
+      <tr>
+        <th scope="col">Type</th>
+        <th scope="col">Desktop</th>
+        <th scope="col">Mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Text</td>
+        <td class="numeric">0.2</td>
+        <td class="numeric">0.2</td>
+      </tr>
+      <tr>
+        <td>XML</td>
+        <td class="numeric">1</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td>Other</td>
+        <td class="numeric">1</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td>Video</td>
+        <td class="numeric">4</td>
+        <td class="numeric">8</td>
+      </tr>
+      <tr>
+        <td>HTML</td>
+        <td class="numeric">3</td>
+        <td class="numeric">14</td>
+      </tr>
+      <tr>
+        <td>Audio</td>
+        <td class="numeric">0.2</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>CSS</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Image</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Script</td>
+        <td class="numeric">30</td>
+        <td class="numeric">30</td>
+      </tr>
+      <tr>
+        <td>Font</td>
+        <td class="numeric">365</td>
+        <td class="numeric">365</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(
+    caption="TODO",
+    sheets_gid="1792973510",
+    sql_file="ttl_by_resource.sql"
+  ) }}</figcaption>
+</figure>
+
+When we look at the entirety of cacheable and non-cacheable resources,
+
+{{ figure_markup(
+  image="cacheable-responses.png",
+  caption="Percent of cacheable vs non-cacheable responses.",
+  description="TODO",
```

Comment:
description="A bar chart showing proportion of cacheable responses. 90.4% of desktop and 89.7% of mobile responses are cacheable.",